pkg/operator/ceph/cluster/nodedaemon/reconcile.go

```diff
@@ -190,8 +190,9 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 		}
 
 		// If the node has Ceph pods we create the daemons
+		var tolerations []corev1.Toleration
```

Comment:
Let's always set this var, otherwise it might be null unexpectedly if `hasCephPods` is false.
```suggestion
		tolerations := uniqueTolerations.ToList()
```

Reply:
done